.github/workflows/publish_image.yaml

```
@@ -0,0 +1,41 @@
+name: Create and publish a Docker image to ghrc.io
+
+on:
+  push
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

```suggestion
  IMAGE_NAME: ${{ github.repository }}
  IMAGE_TAG: ${{ github.sha }}
```
can the tag be [the sha1 from the environment](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)?